internal/pkg/agent/cmd/common.go

```diff
@@ -43,7 +43,7 @@ func NewCommandWithArgs(args []string, streams *cli.IOStreams) *cobra.Command {
 
 	// Init version information contained in package version file
 	err := version.InitVersionError()
-	if err != nil {
+	if isOtel := len(args) > 1 && args[1] == "otel"; !isOtel && err != nil {
```

Comment:
Should we even try to initialize the version (previous line) if we are in `otel` mode?